### ISSUE TITLE
Invalid assumptions about the layout of node_modules causes errors with npm@3, and with npm < 3 after npm dedupe

### ISSUE THREAD

**Issue by aredridel** [Strider-CD/strider#844]
Because strider requires deep modules across module boundaries as in `lib/routes/admin/plugins/get_plugin_list.js`, lines like

```
var client = require('strider-cli/node_modules/strider-ecosystem-client');
```
throw an error.

A better approach might be to have `strider-cli` export that module itself, so you can `require('strider-cli').someProperty` to access that submodule's interface, or to depend on `strider-ecosystem-client` directly.

**Comment by knownasilya** [comment]
@aredridel could you tackle this as a PR?

**Comment by aredridel** [comment]
Possibly, if I get time!

**Comment by knownasilya** [comment]
Fixed by be36b4cce8e83f80a69d59e4ebd3e94fa2764a41
